lib/services/platform-service.ts

```diff
@@ -175,9 +175,15 @@ export class PlatformService implements IPlatformService {
 			platformData.platformProjectService.prepareProject().wait();
 
 			// Process node_modules folder
-			this.$pluginsService.ensureAllDependenciesAreInstalled().wait();
-			let tnsModulesDestinationPath = path.join(platformData.appDestinationDirectoryPath, constants.APP_FOLDER_NAME, PlatformService.TNS_MODULES_FOLDER_NAME);
-			this.$broccoliBuilder.prepareNodeModules(tnsModulesDestinationPath, this.$projectData.projectDir, platform, lastModifiedTime).wait();
+			try {
+				this.$pluginsService.ensureAllDependenciesAreInstalled().wait();
+				let tnsModulesDestinationPath = path.join(platformData.appDestinationDirectoryPath, constants.APP_FOLDER_NAME, PlatformService.TNS_MODULES_FOLDER_NAME);
+				this.$broccoliBuilder.prepareNodeModules(tnsModulesDestinationPath, this.$projectData.projectDir, platform, lastModifiedTime).wait();
+			} catch(error) {
+				this.$logger.debug(error);
+				this.$logger.error("Processing tns core modules failed.");
```

Comment:
This is user visible error. Consider at least showing the users the actual error. This way, they have a chance to understand what went wrong.